apps/src/javalab/JavalabEditor.jsx

```diff
@@ -10,7 +10,7 @@ import {EditorState} from '@codemirror/state';
 const style = {
   editor: {
     width: '100%',
-    height: 600,
+    height: 400,
```

Comment:
I only changed this so everything would fit on my small laptop screen. Happy to change it back.